Pod/Classes/Cards/Default Card Types/ChinaUnionPay.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  ChinaUnionPay.swift
+//  Pods
+//
+//  Created by Shiyuan Jiang on 3/30/16.
+//
+//
+
+import UIKit
+
+public struct ChinaUnionPay: CardType {
```

Comment:
For code completion reasons, we should have all public / internal stuff documented, even if it is self-descriptive.